### ISSUE TITLE
Question with gender classification in training and inference

### ISSUE THREAD

**Issue by jimmy-walker** [oarriaga/face_classification#84]
Hello, It's so kind for you to provide this good project. I really appreciate.

I just wanna ask a question as followed:
Why you use "haarcascade_frontalface_default.xml" to detect face in inference stage(image_emotion_gender_demo.py), however you don't use "haarcascade_frontalface_default.xml" to detect face in training stage(train_gender_classifier.py), you just write your own augment script(data_augmentation.py).
**In short, I am wondering why you preprocess the image in different way between training and inference.**
**Why not use "haarcascade_frontalface_default.xml" in training stage(train_gender_classifier.py)?**

Looking forward to reply, Thanks.

**Comment by oarriaga** [comment]
Training data has only cropped faces, and I wanted to provide a complete detection + classification pipeline in the project; therefore, the two stages.
